ranger/container/directory.py

```diff
@@ -252,6 +252,19 @@ def hidden_filter_func(fobj):
                         return False
                 return True
             filters.append(hidden_filter_func)
+        if self.settings.global_inode_type_filter and not self.inode_type_filter:
+            # pylint: disable=too-many-boolean-expressions
+            def hidden_inode_filter_func(obj):
+                g_filter = self.settings.global_inode_type_filter
+                if (
+                        ('d' in g_filter and obj.is_directory) or
```

Comment:
Also, it's almost the same function as the one used in `commands.py`, in `filter_inode_type.execute`. Maybe we can reuse it and not duplicate this code? Especially since that one uses the named constants instead of the magic strings (`self.FILTER_DIRS` vs `'d'`).